client/src/entry/analysis/AnalysisRouter.js

```diff
@@ -105,6 +105,8 @@ export const getAnalysisRouter = (Galaxy) => {
             "(/)collection(/)edit(/)(:collection_id)": "show_collection_edit_attributes",
             "(/)datasets/error": "show_dataset_error",
             "(/)datasets(/)(:dataset_id)/details": "show_dataset_details",
+            // legacy url for older links
+            "(/)datasets(/)(:dataset_id)/show_params": "show_dataset_details",
```

Comment:
should we just change original `(/)datasets(/)(:dataset_id)/details` to `(/)datasets(/)(:dataset_id)/show_params` so we don't have to keep redirect?

Reply:
It's not a redirect, and I think `/details` is more accurate ?

Reply:
> It's not a redirect, and I think `/details` is more accurate ?

yes, it's definitely more accurate, I just thought it might help if we reduce the number of endpoint. But I guess it's better to keep keep this for better understanding and mark `show_params` as deprecated